src/components/labelled-date-input/LabelledDateInput.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import DateTime from 'react-datetime'
+import 'react-datetime/css/react-datetime.css'
+import moment from 'moment'
+import 'moment/locale/ru'
+import './LabelledDateInput.css'
+import LabelledInput from '../labelled-input'
+
+const LabelledDateInput = ({ label, ...other }) => (
```

Comment:
lets simply pass all props down to the LabelledInput
```suggestion
const LabelledDateInput = (props) => (
```